src/groupByKey.ts

```diff
@@ -2,6 +2,15 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+export function groupByKey(items: Item[], field: keyof Item) :Item {
+  const result: Item = {};
```

Comment:
```suggestion
  const result: Item = {};
```
The result should be not of an 'Item' type. Each key should be an array of 'items' (not 'any')

Create the interface
```
{
  [key: string] : Item[],
}
```
and use it as a result type
Also, `result` is a bad naming